pallets/exchange/src/weights.rs

```diff
@@ -60,63 +60,63 @@ pub trait WeightInfo {
 pub struct HackHydraWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for HackHydraWeight<T> {
 	fn known_overhead_for_on_finalize() -> Weight {
-		(13_726_000 as Weight)
+		(14_683_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 	}
 	fn sell_intention() -> Weight {
-		(85_164_000 as Weight)
+		(119_553_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(6 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	fn buy_intention() -> Weight {
-		(85_048_000 as Weight)
+		(120_097_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(6 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	fn on_finalize(t: u32, ) -> Weight {
-		(21_475_000 as Weight)
-			// Standard Error: 36_000
-			.saturating_add((156_991_000 as Weight).saturating_mul(t as Weight))
+		(49_006_000 as Weight)
```

Comment:
Is this on the reference machine?

Reply:
yes. always as it is generated by the benchbot